packages/integration-tests/suites/replay/privacy/test.ts

```diff
@@ -0,0 +1,226 @@
+import { expect } from '@playwright/test';
+import { EventType } from '@sentry-internal/rrweb';
+import type { RecordingEvent } from '@sentry/replay';
+
+import { sentryTest } from '../../../utils/fixtures';
+import { envelopeRequestParser } from '../../../utils/helpers';
+import { waitForReplayRequest } from '../../../utils/replayHelpers';
+
+sentryTest('should have the correct default privacy settings', async ({ getLocalTestPath, page }) => {
+  // For this test, we skip all bundle tests, as we're only interested in Replay being correctly
+  // exported from the `@sentry/browser` npm package.
+  if (process.env.PW_BUNDLE && process.env.PW_BUNDLE.startsWith('bundle_')) {
+    sentryTest.skip();
+  }
```

Comment:
m: Let's also test this against the bundles, just to be sure we don't mess things up with minification/mangling. 

```suggestion
  // Replay bundles are es6 only
  if (process.env.PW_BUNDLE && process.env.PW_BUNDLE.startsWith('bundle_es5')) {
    sentryTest.skip();
  }
```